generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/FixedField.java

```diff
@@ -23,7 +23,7 @@
     private Object current = NOT_ITERATED;
     private FieldSpec fieldSpec;
 
-    FixedField(
+    public FixedField(
```

Comment:
Made public so the `ReductiveDecisionTreeWalkerTests` type can create an instance of a `FixedField`, maybe there is a better way?

Reply:
Resolved by way of using `mockito` mocks